bootcamp/contrib/student_15.py

```diff
@@ -0,0 +1,12 @@
+def CaseConversion(string):
+    """
+    Change Upper Case to Lower Case; Vise versa
+    """
+    CaseDict = {
+        'a': 'A', 'c': 'C', 'g': 'G', 't': 'T',
+        'A': 'a', 'C': 'c', 'G': 'g', 'T': 't'
+    }
+    conversionString = ''
+    for base in string:
+        conversionString = conversionString + CaseDict[base]
```

Comment:
While this works fine, a cleaner approach would be to simply swap the case of each letter using something like [`islower()`](https://docs.python.org/3/library/stdtypes.html?highlight=islower#str.islower).